app/controllers/banneduser_experiment_controller.py

```diff
@@ -279,6 +280,15 @@ def _is_tempban(self, modaction):
         """
         return modaction["action"] == "banuser" and "days" in modaction["details"]
 
+    def _is_tempban_edit(self, modaction):
+        """Return true if an admin action is a temporary ban edit.
+
+        For temporary bans, we expect the string "changed to" in `details`. E.g. "changed to 1 days".
+
+        This calls `_is_tempban`, even if redundant, as to be unambiguous and not assume that modaction is already a tempban.
+        """
+        return self._is_tempban(modaction) and "changed to" in modaction["details"]
```

Comment:
may need an update:

```suggestion
        return self._is_tempban(modaction) and "changed to" in modaction.details
```

based on changes in #83